doc/ChangeLog

```diff
@@ -1,4 +1,265 @@
 ===============================================================
+Tag name: ctsm5.3.014
+Originator(s): multiple (see contributors below)
+Date: Mon 02 Dec 2024 09:50:09 AM MST
+One-line Summary: Bring in several fixes for testing in the previous cesm3_0_beta03/04 tags
+
+Purpose and description of changes
+----------------------------------
+
+Fix a list of issues mostly for testing that came in with the cesm3_0_beta03 and cesm3_0_beta04 tags
+for the science "chill" deadline bringing in the baseline science capabilities needed for the cesm3_0 release.
+
+List of things:
+
+- New polarcap grid
+- Fix use_init_interp for several resolutions that had
+- Remove clm5_1 physics option
+- Newly needed surface datasets added to auto build in Makefile for mksurfdata_esmf
+- Fix several individual tests that were failing
+- Update to submodules to ones roughly based on cesm3_0_beta04
+- Add graceful error checking to the FATES parameter file tests that will get it working in some cases
+
+Contributors
+------------
+    @slevis-lmwg @ekluzek @adamrher
```

Comment:
I would add @samsrabin due to his contribution in #2801

Reply:
Done